### ISSUE TITLE
Make the path diagrams useful 

### ISSUE THREAD

**Issue by kylehamilton** [kylehamilton/lavaan.shiny#3]
Allow for additional settings to be changed, allow for the images to be downloaded, make them not ugly

**Comment by VectorPosse** [comment]
A lot of the plot is getting cut off at the bottom and top of the diagram. Is that what you mean by "ugly"?

**Comment by kylehamilton** [comment]
Yeah also I thought it would be a good idea to let the user change the settings for semPlot but that's also a concern. Is the problem you're describing in the current CRAN version or the development version?

**Comment by VectorPosse** [comment]
I'm using the CRAN version.

After playing around with the semPlot package directly, I can make the
"cutoff" problem go away with the mar parameter, but there seems to be no
way to know in advance what the values of mar should be. So I think you'd
have no choice but to allow the user to tweak the parameters themselves.
(It would be super cool to have some margin "buttons" on all four sides
that one could click and watch the graph respond by nudging the margins in
or out.)

By the way, thanks for creating this awesome tool.
